app/models/organisation.rb

```diff
@@ -2,24 +2,24 @@
 require 'string'
 
 class Organisation < BaseOrganisation
-
+  has_many :events
   has_many :volunteer_ops
   has_many :users
-  has_many :edits, class_name: 'ProposedOrganisationEdit', :dependent => :destroy
+  has_many :edits, class_name: 'ProposedOrganisationEdit', dependent: :destroy
 
   accepts_nested_attributes_for :users # TODO check if needed
 
   scope :order_by_most_recent, -> { order('organisations.updated_at DESC') }
-  scope :not_null_email, lambda {where("organisations.email <> ''")}
+  scope :not_null_email, -> { where("organisations.email <> ''") }
```

Comment:
these syntax improvements are lovely, but they do make reviewing the PR a little harder - would be great to get these sorts of commits into a refactoring PR so that we could merge them separately (and faster) and avoid the chance of merge conflicts.  I know it's not such a nice work flow, but something to think about ...

Reply:
You are totally right! would you like me to revert those changes or just not to add any more changes related to refactoring?